apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -309,18 +306,101 @@ const TutorialExplorer = React.createClass({
   }
 });
 
+
+function getFilters(robotics) {
+  const filters = [
+    { name: "grade",
+      text: "Grades",
+      entries: [
+        {name: "pre", text: "Pre-reader"},
+        {name: "2-5", text: "Grades 2-5"},
+        {name: "6-8", text: "Grades 6-8"},
+        {name: "9+", text: "Grades 9+"}]},
+    { name: "teacher_experience",
+      text: "Educator's experience",
+      entries: [
+        {name: "beginner", text:"Beginner"},
+        {name: "comfortable", text:"Comfortable"}]},
+    { name: "student_experience",
+      text: "Student's experience",
+      entries: [
+        {name: "beginner", text: "Beginner"},
+        {name: "comfortable", text: "Comfortable"}]},
+    { name: "platform",
+      text: "Classroom technology",
+      entries: [
+        {name: "computers", text: "Computers"},
+        {name: "android", text: "Android"},
+        {name: "ios", text: "iPad/iPhone"},
+        {name: "no-internet", text: "Poor or no internet"},
+        {name: "no-computers", text: "No computers or devices"}]},
+    { name: "subject",
+      text: "Topics",
+      entries: [
+        {name: "science", text: "Science"},
+        {name: "math", text: "Math"},
+        {name: "history", text: "Social Studies"},
+        {name: "la", text: "Language Arts"},
+        {name: "art", text: "Art, Media, Music"},
+        {name: "cs-only", text: "Computer Science only"}]},
+    { name: "activity_type",
+      text: "Activity type",
+      entries: [
+        {name: "online-tutorial", text: "Self-led tutorials"},
+        {name: "lesson-plan", text: "Lesson plan"}]},
+    { name: "length", text: "Length",
+      entries: [
+        {name: "1hour", text: "One hour"},
+        {name: "1hour-follow", text: "One hour with follow-on"},
+        {name: "few-hours", text: "A few hours"}]},
+    { name: "programming_language",
+      text: "Language",
+      entries: [
+        {name: "blocks", text: "Blocks"},
+        {name: "typing", text: "Typing"},
+        {name: "other", text: "Other"}]}];
+
+  const initialFilters = {
+    teacher_experience: ["beginner"],
+    student_experience: ["beginner"]
+  };
+
+  const hideFilters = {
+    activity_type: ["robotics"]
+  };
+
+  if (robotics) {
+    for (var index = 0; index < filters.length; index++) {
+      var filterGroup = filters[index];
+      if (filterGroup.name === "activity_type") {
+        filterGroup.entries = [{name: "robotics", text: "Robotics"}];
+        filterGroup.display = false;
+      }
+    }
```

Comment:
What you have is fine (though I'd use let instead of var). Could do this somewhat more concisely (tho this is probably me being somewhat nitty/personal preference):
```js
filters.forEach(filterGroup => {
  if (filterGroup.name === "activity_type") {
    filterGroup.entries = [{name: "robotics", text: "Robotics"}];
    filterGroup.display = false;
 }
})